cfgov/cfgov/urls.py

```diff
@@ -251,6 +233,37 @@
     # Report redirects
     url(r'^reports/(?P<path>.*)$', RedirectView.as_view(url='/data-research/research-reports/%(path)s', permanent=True)),
 ]
+
+if settings.DEBUG:
```

Comment:
It might be a little more transparent to put this logic in settings.base?

like,

`ALLOW_ADMIN_URL = DEBUG or os.environ.get('ALLOW_ADMIN_URL', False)`

This also assumes false if ALLOW_ADMIN_URL is unset, which seems safer.